pkg/api/errors/cryptography.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+import (
+	"fmt"
+	"net/http"
+
+	grpcCodes "google.golang.org/grpc/codes"
+
+	kiterrors "github.com/dapr/kit/errors"
+)
+
+const (
+	ErrCryptoProviderNotConfigured = "NOT_CONFIGURED"
+	ErrCryptoProviderNotFound      = "NOT_FOUND"
+	ErrCryptoNameEmpty             = "NAME_EMPTY"
+	ErrCryptoGetKey                = "GET_KEY"
+	ErrCryptoEncryptOperation      = "ENCRYPT"
+	ErrCryptoDecryptOperation      = "DECRYPT"
+	ErrCryptoKeyOperation          = "KEY"
+	ErrCryptoVerifySignature       = "VERIFY_SIGNATURE"
+)
+
+const ResourceType = "Cryptography"
+
+func CryptoNotConfigured(name string) error {
+	message := "crypto providers not configured"
+	return kiterrors.NewBuilder(
+		grpcCodes.Internal,
+		http.StatusInternalServerError,
+		message,
+		"ERR_CRYPTO_PROVIDERS_NOT_CONFIGURED",
+	).
+		WithErrorInfo(kiterrors.CodePrefixCryptography+ErrCryptoProviderNotConfigured, nil).
+		WithResourceInfo(ResourceType, name, "", message).
+		Build()
+}
+
+func CryptoNotFound(name string) error {
```

Comment:
same here. There is a generic `NotFound` you can use here: https://github.com/dapr/dapr/blob/master/pkg/api/errors/errors.go#L11